pkg/util/gce/gce.go

```diff
@@ -88,6 +88,10 @@ func getResponse(url string) (string, error) {
 		return "", fmt.Errorf("GCE hostname, error reading response body: %s", err)
 	}
 
+	if len(all) <= 0 {
```

Comment:
Could you add a quick comment explaining why we have to check for an empty body ?